sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -95,26 +95,45 @@ export class KeyVaultAccessControlClient {
       ? challengeBasedAuthenticationPolicy(credential)
       : signingPolicy(credential);
 
-    const internalPipelineOptions = {
-      ...pipelineOptions,
-      ...{
-        loggingOptions: {
-          logger: logger.info,
-          logPolicyOptions: {
-            allowedHeaderNames: [
-              "x-ms-keyvault-region",
-              "x-ms-keyvault-network-info",
-              "x-ms-keyvault-service-version"
-            ]
-          }
-        }
+    const internalPipelineOptions: InternalPipelineOptions = {
+      // coreHttp.PipelineOptions has "serviceVersion", but InternalPipelineOptions doesn't. Is that expected?
+      // serviceVersion: pipelineOptions.serviceVersion,
+
+      httpClient: pipelineOptions.httpClient,
+      retryOptions: pipelineOptions.retryOptions,
+      proxyOptions: pipelineOptions.proxyOptions,
+      keepAliveOptions: pipelineOptions.keepAliveOptions,
+      redirectOptions: pipelineOptions.redirectOptions,
+      userAgentOptions: pipelineOptions.userAgentOptions,
+      loggingOptions: {
+        logger: logger.info
+
+        // "logPolicyOptions" is not a valid parameter of "loggingOptions". Is that expected?
```

Comment:
I'm confused by this, since this is the definition I see:

```ts
/**
 * Defines options that are used to configure internal options of
 * the HTTP pipeline for an SDK client.
 */
export interface InternalPipelineOptions extends PipelineOptions {
  /**
   * Options to configure API response deserialization.
   */
  deserializationOptions?: DeserializationOptions;

  /**
   * Options to configure request/response logging.
   */
  loggingOptions?: LogPolicyOptions;

  /**
   * Configure whether to decompress response according to Accept-Encoding header (node-fetch only)
   */
  decompressResponse?: boolean;

  /**
   * Send JSON Array payloads as NDJSON.
   */
  sendStreamingJson?: boolean;
}
```

Reply:
I've made this work! and I wrote tests. Thank you!